### PR TITLE
Bumps OpenAPIKit to version 3.0.0-alpha.7 fixing an issue with trailing slashes.

### PR DESCRIPTION
**Pull request by miguel-arrf**
### Motivation
As discussed in #51, there was an issue with paths containing trailing slashes. The problem was identified in the [OpenAPIKit](https://github.com/mattpolzin/OpenAPIKit) dependency. It was fixed in the PR https://github.com/mattpolzin/OpenAPIKit/pull/274.

### Modifications
This PR updates the OpenAPIKit dependency version from `3.0.0-alpha.6` to `3.0.0-alpha.7` ([available here](https://github.com/mattpolzin/OpenAPIKit/releases/tag/3.0.0-alpha.7)), which includes the mentioned PR.

### Result
If a path ends with a trailing slash, such as `foo/bar/`, the generated functions will now correctly call the path `foo/bar/` instead of (as previously) `foo/bar`.

### Test Plan
Replaced an endpoint in the reference test to use a path with a trailing slash.

### Resolves
- Resolves https://github.com/apple/swift-openapi-generator/issues/51

### CODE REVIEW

**Comment by simonjbeaumont** [comment]
@swift-server-bot add to allowlist

**Comment by czechboy0** [comment]
You could just change one of the paths in the integration test to end with a slash instead of adding another path+operation. But it'd be good for our tests to include this case.

**Review by simonjbeaumont — CHANGES_REQUESTED** [review]
Thanks for the fix. Based on the discussion here and on the issue, I'll mark this as request changes, until we get the test changed.

**Review by czechboy0 — CHANGES_REQUESTED** [review]
Thanks @miguel-arrf for updating the tests, but I don't think we need a new operation for it, as those are pretty expensive to maintain (as you just saw when adding a new one), so we add new ones sparingly, only when we can't extend existing ones. An easier alternative is to modify the path of one of the existing operations, and update the corresponding test.

**Comment by miguel-arrf** [comment]
@czechboy0 Ah, I see! Thank you, I'll update it accordingly :)!

**Comment by miguel-arrf** [comment]
(I still have some more tests to change, because of the Server!)

File: Tests/PetstoreConsumerTests/Test_Client.swift [hunk 1]
```diff
@@ -344,7 +344,7 @@ final class Test_Client: XCTestCase {
     func testProbe_204() async throws {
         transport = .init { request, baseURL, operationID in
             XCTAssertEqual(operationID, "probe")
-            XCTAssertEqual(request.path, "/probe")
+            XCTAssertEqual(request.path, "/probe/")
```
Comment (czechboy0) [inline Tests/PetstoreConsumerTests/Test_Client.swift:347]: Great, thanks for updating the `PetstoreConsumerTests` target as well – this is the important bit that verifies that the trailing slash is actually passed to the transport, and not stripped between OpenAPIKit and the generated code.

**Review by czechboy0 — APPROVED** [review]
Looks great, thank you, @miguel-arrf! 🙏

**Comment by simonjbeaumont** [comment]
@miguel-arrf Looks great!

Would you like to update the PR description given you've gone to the effort of sorting the testing out?

> ⚠️ No steps have been taken yet to test this PR. The existing tests should be updated to include paths with trailing slashes.

**Review by simonjbeaumont — APPROVED** [review]

**Comment by miguel-arrf** [comment]
Thanks @simonjbeaumont ! ☺️

**Pull request merged**